accounts_backend/views/mozilla_accounts_multi_service_dau.view.lkml

```diff
@@ -0,0 +1,80 @@
+view: mozilla_accounts_multi_service_dau {
+
+  derived_table: {
+
+    sql:
+    SELECT
+      DATE(e.submission_timestamp) as submission_date,
+      e.metrics.string.account_user_id_sha256 AS user_id,
+      ARRAY_AGG(DISTINCT c.name IGNORE NULLS ORDER BY c.name) as service_names,
+    FROM
+      `mozdata.accounts_backend.accounts_events` AS e
+    JOIN
+      `mozdata.accounts_db.fxa_oauth_clients` AS c
+    ON e.metrics.string.relying_party_oauth_client_id = c.id
+    WHERE c.name IN
+        (
+        'AMO Stage',
+        'Firefox',
+        'Firefox iOS',
+        'Firefox for Android',
+        'Firefox Relay',
+        'MDN Plus',
+        'Mozilla VPN',
+        'Mozilla Monitor',
+        'Mozilla IAM',
+        'Mozilla Social',
+        'Mozilla Support',
+        'Mozilla Hubs',
+        'Pontoon',
+        'Pocket',
+        'Thunderbird Add-ons'
```

Comment:
- Probably want `Add-ons` instead of `AMO Stage`.
- Maybe also `MDN Plus App`?
- It'd be nice if these were in alphabetical order (it's close but not quite).

Reply:
Also, I'd worry about how this condition will be maintained over time, as I think it'd be easy for us to miss when new services appear that should be added here or remember that this specific LookML needs to be updated.

A less precise approach of not filtering on specific service names would avoid that problem, at the cost of skewing the multi-service user counts (looking at the last week of data, not filtering on specific services would increase the multi-service user counts by ~2.3%).

Reply:
I was worried about overwhelming the graph with permutations of naming combinations, but I forgot I sorta by-passed this in the first place by adding a filter on the graph for having e.g user counts > 500, so the filtering is not really necessary.